### PR TITLE
fix: updated everything eslint

### PR DESCRIPTION
**Pull request by IgorSzyporyn**
And exluded "app" and "examples" from react hook dependencies

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    // "inlineSources": true,                       /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
+
+    /* Experimental Options */
+    // "experimentalDecorators": true,              /* Enables experimental support for ES7 decorators. */
+    // "emitDecoratorMetadata": true,               /* Enables experimental support for emitting type metadata for decorators. */
+
+    /* Advanced Options */
+    "skipLibCheck": true /* Skip type checking of declaration files. */,
+    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
+  }
```
Comment (ndelangen) [inline tsconfig.json:71]: remove all comments please

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
-  "eslint.enable": true,
-  "deepscan.enable": true
-}
\ No newline at end of file
+  "deepscan.enable": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```
Comment (ndelangen) [inline .vscode/settings.json:3]: Is this needed?

**Review by ndelangen — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (IgorSzyporyn) [reply .vscode/settings.json:3]: Well I just saw expectations of prettier to work in a test file and the presence of a .vscode folder so I added it - it is not required no

File: tsconfig.json (hunk 1 above)
Comment (IgorSzyporyn) [reply tsconfig.json:71]: Will do

**Review by ndelangen — APPROVED** [review]

**Pull request merged**